cmd/crank/update.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/alecthomas/kong"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	v1 "github.com/crossplane/crossplane/apis/pkg/v1"
+	typedclient "github.com/crossplane/crossplane/internal/client/clientset/versioned/typed/pkg/v1"
+
+	// Load all the auth plugins for the cloud providers.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+
+// updateCmd updates a package.
+type updateCmd struct {
+	Configuration updateConfigCmd   `cmd:"" help:"Update a Configuration package."`
+	Provider      updateProviderCmd `cmd:"" help:"Update a Provider package."`
+}
+
+// Run runs the update cmd.
+func (c *updateCmd) Run(b *buildChild) error {
+	return nil
+}
+
+// updateConfigCmd updates a Configuration.
+type updateConfigCmd struct {
+	Name string `arg:"" help:"Name of Configuration."`
+	Tag  string `arg:"" help:"Updated tag for Configuration package."`
+
+	RevisionHistoryLimit *int64   `short:"rl" default:"-1" help:"Revision history limit."`
+	ManualActivation     *bool    `short:"m" help:"Enable manual revision activation policy."`
```

Comment:
@weastel my apologies, it appears that pointers are not valid types in the `kong` parse tree, which is what is currently causing the unit test failures right now. Using a `default:"-1"` value as you have done here is not a bad idea at all. Another thing I could imagine trying is using a hook. However, for now would you be okay with just doing `Name` and `Tag` and we can revisit any of the flags in a follow-up if needed?